sw/device/silicon_creator/lib/drivers/spi_device.c

```diff
@@ -48,10 +499,32 @@ void spi_device_init(void) {
   abs_mmio_write32(kBase + SPI_DEVICE_JEDEC_ID_REG_OFFSET, reg);
 
   // Configure READ_JEDEC_ID command (CMD_INFO_3).
-  reg = bitfield_field32_write(0, SPI_DEVICE_CMD_INFO_3_OPCODE_3_FIELD,
-                               kSpiDeviceCmdReadJedecId);
-  reg = bitfield_bit32_write(reg, SPI_DEVICE_CMD_INFO_3_VALID_3_BIT, true);
-  abs_mmio_write32(kBase + SPI_DEVICE_CMD_INFO_3_REG_OFFSET, reg);
+  cmd_info_set((cmd_info_t){
+      .reg_offset = SPI_DEVICE_CMD_INFO_3_REG_OFFSET,
+      .op_code = kSpiDeviceCmdReadJedecId,
+      .address = false,
+      .dummy_cycles = 0,
+  });
+  // Configure READ_SFDP command (CMD_INFO_4).
```

Comment:
Just a comment: I wonder if we ought to have regtool generate constants / macros that associate cmd_info slots with the specific hardware-handled commands.

Reply:
Good point, I think the current design let's us customize the actual opcode but you have to write it to the correct cmd_info register. @eunchan I'll add a link to this comment in #11740.